src/v/redpanda/application.cc

```diff
@@ -1236,6 +1235,7 @@ application::set_proxy_client_config(ss::sstring name, std::any val) {
 }
 
 void application::start(::stop_signal& app_signal) {
+    wire_up_services();
```

Comment:
The conceptual difference between "wire up" and "start" is that the former just instantiates a bunch of objects with references to one another, whereas the latter kicks off all the background fibers.

I'm not sure I have a super strong reason, but gut-feel I kind of liked that separation.

Reply:
Yeah, I started going down this route because these aren't being used in any meaningfully different way.

My actual motivation for simplifying the testing startup sequence is that I intend on changing the sequence to be 1. start the subsystems required to bootstrap, 2. start everything else. Hiding that dance under a general `start()` method was just a way to put less onus on test fixtures and other users of the `application`.

That said, maybe I'll keep the separation in still, but include a new `wire_up_and_start()` or somesuch.